tests/Agent/IntegrationTests/IntegrationTests/BasicInstrumentation/WebApiAsyncForceNewTransactionTests.cs

```diff
@@ -174,10 +174,10 @@ public void Test()
 
 
                 //Cannot determine async-fire and forget because it will probably throw an error, but can determine what it starts with
-                () => Assert.NotEmpty(metrics.Where(x => x.MetricSpec.Name.StartsWith("WebTransaction/FireAndForgetTests/AF-", StringComparison.OrdinalIgnoreCase))),
+                () => Assert.True(metrics.Where(x => x.MetricSpec.Name.StartsWith("WebTransaction/FireAndForgetTests/AF-", StringComparison.OrdinalIgnoreCase)).Count() > 0),
```

Comment:
These should use `Assert.Contains()` instead of `Assert.True()`:
```
() => Assert.Contains(metrics, x => x.MetricSpec.Name.StartsWith("WebTransaction/FireAndForgetTests/AF-", StringComparison.OrdinalIgnoreCase)),
```